src/aioftp/client.py

```diff
@@ -137,7 +140,13 @@ def __init__(
         self.parse_list_line_custom = parse_list_line_custom
         self.parse_list_line_custom_first = parse_list_line_custom_first
         self._passive_commands = passive_commands
-        self._open_connection = partial(open_connection, ssl=self.ssl, **siosocks_asyncio_kwargs)
+        self._open_connection = partial(
+            open_connection,
+            ssl=ssl if not explicit_tls else None,
+            **siosocks_asyncio_kwargs,
+        )
```

Comment:
I think `ssl` and `explicit_tls` can't be passed together, so we need to check this and raise an error in this case